common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelInput.java

```diff
@@ -116,51 +124,16 @@ public MLUpdateModelInput(StreamInput in) throws IOException {
                 this.deploySetting = new MLDeploySetting(in);
             }
         }
+        if (streamInputVersion.onOrAfter(MLRegisterModelInput.MINIMAL_SUPPORTED_VERSION_FOR_MODEL_INTERFACE)) {
+            if (in.readBoolean()) {
+                modelInterface = in.readMap(StreamInput::readString, StreamInput::readString);
+            }
+        }
     }
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject();
-        builder.field(MODEL_ID_FIELD, modelId);
```

Comment:
Why remove these lines?

Reply:
Add java doc to this method and add this comment. It's easy to forget why

Reply:
Added back